src/components/SearchComponents/Search.page.test.tsx

```diff
@@ -41,22 +41,32 @@ const validQuery: SearchQuery = {
 
 const setSearchStartTime = jest.fn();
 
-const props: SearchProps = {
+export const props: SearchProps = {
   status: "not-asked",
   searchStartTime: 1,
   setSearchStartTime: setSearchStartTime,
   results: [],
   query: validQuery,
   setQuery: jest.fn(),
-  keyStageFilters: [
-    {
-      slug: "ks1",
-      title: "Key-stage 1",
-      shortCode: "KS1",
-      onChange: jest.fn(),
-      checked: false,
-    },
-  ],
+  searchFilters: {
+    keyStageFilters: [
+      {
+        slug: "ks1",
+        title: "Key-stage 1",
+        shortCode: "KS1",
+        onChange: jest.fn(),
+        checked: false,
+      },
+    ],
+    subjectFilters: [
```

Comment:
Have you added a test in for subjectFilters specifcally in this suite?

Reply:
No will add one for subject onClick